_posts/2023-01-03-state-of-devops.md

```diff
@@ -0,0 +1,55 @@
+
+Well, to take a real-life example, I bought a truck a few years back because I was about to buy a camper and would need to haul it places. So, bought the truck, drove it off the lot and to home. Seeing as it was a newer truck, it was actually a nicer drive than my old car. Got it home and parked it just fine. However, the next day, I needed to drive it to the office which had some minor struggles. I was used to the smaller space my car took and the fact I didn't need quite the same clearance room to change lanes and I didn't want to be "that guy with a truck" all over the interstate. So it took me about another minute to get to the office, not bad.
+
+Then I had to park the thing. The parking lot at the office was a bit tight, but I was earlier than most, so I just parked a little further back where I wasn't surrounded, but it added another minute or two of walking to my desk. So, nice, smoother ride that didn't sometimes die randomly for the cost of about 3 minutes longer door to door. Seems like a good trade-off of reliability for speed.
+
+However, I bought a camper a few month later. If you have never tried to pull 20' of camper behind a vehicle (without swaybars), it's a new experience. Instead of the normal 2ish hours home, it took nearly 3 of going slower, stopping and checking to make sure things were attached, and finding a gas station I could pull through with the camper on it. My normal efficiency of driving plummeted by 50% while bringing the camper back. 
+
+The ability to camp without a tent was my goal, however, and so with a lot more experience of driving the truck and the camper places I got faster and better at it. At this point, I am pretty confident of driving the camper wherever and enjoy camping again. Thus, my goal of enjoying and camping more often and being efficient have been met, but there was a solid 18 months of learning to drive the truck differently (especially in the snow up here in Wisconsin), pulling a trailer, and generally adapting to an entire new way to go place.
+
+This is how the J-Curve works, not only did I need to learn a new tool (driving the truck) and a new technique (moving the camper), I also had to get over the fear of sliding the camper into other cars, thinking ahead on my turns, and planning places to take the camper. However, today in 2023, I can take the camper many places and my real concern is did I remember to pack everything and can drive my truck pretty efficiently. As you look to adapt to new tools to ensure reliability and the ability to pull your organization to your vision, just realize there will be a time they slow down and learn and that the whole organization will need to adapt to a new way of thinking. You can't just jump from a little car to a truck-and-trailer nor can you jump from a classic software desire to a powerful, robust, and scalable one without several months of adapting to it.
```

Comment:
This sentence is clunky: "However, today in 2023, I can take the camper many places and my real concern is did I remember to pack everything and can drive my truck pretty efficiently."  Not sure what you're trying to say here. 

Typo: "classic software design", maybe?